src/makeRobotAccountant.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function getSum(y) {
```

Comment:
```suggestion
  return function getSum(firstNumber) {
```
Use more informative names for variables